jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/execution/sam/LambdaSamRunConfigurationProducerTest.kt

```diff
@@ -43,16 +43,16 @@ class LambdaLocalRunConfigurationProducerTest {
         runInEdtAndWait {
             val runConfiguration = createRunConfiguration(lambdaMethod)
             assertThat(runConfiguration).isNotNull
-            val configuration = runConfiguration?.configuration as LambdaLocalRunConfiguration
+            val configuration = runConfiguration?.configuration as SamRunConfiguration
             assertThat(configuration.getHandler()).isEqualTo("com.example.LambdaHandler::handleRequest")
             assertThat(configuration.name).isEqualTo("com.example.LambdaHandler::handleRequest")
         }
     }
 
     @Test
-    fun invalidLambdaIsNotCrated() {
+    fun invalidLambdaIsNotCreated() {
         val psiClass = projectRule.fixture.openClass(
-            """
+                """
```

Comment:
where's this weird indenting coming from?

Reply:
Didn't notice the IDE auto formatting the indent. Corrected it in the new commit.